staff/ana-sanjuan/cv/index.html

```diff
@@ -5,121 +5,115 @@
-    </section>
-
-    <hr>
-
-    <section>
-        <h2>Skills</h2>
-        <ol>
-            <li><b>Programming Languages:</b> lorem, lorem, lorem and lorem.</li>
-            <li><b>Databases:</b> lorem, lorem, lorem and lorem.</li>
-            <li><b>Platforms:</b> lorem, lorem, lorem and lorem.</li>
-            <li><b>Version Control:</b> lorem, lorem, lorem and lorem.</li>
-            <li><b>Operating Systems:</b> lorem, lorem, lorem and lorem.</li>
-            <li><b>Other Tools:</b> lorem, lorem, lorem and lorem.</li>
-            <li><b>Soft Skills:</b> lorem, lorem, lorem and lorem.</li>
-        </ol>
-    </section>
-
-    <hr>
-
-    <section>
-        <h2>Work Experience</h2>
-        <table>
-            <tr>
-                <th>Years</th>
-                <th>Company Name</th>
-                <th>Location</th>
-                <th>Position title</th>
-                <th>Responsabilities</th>
-            </tr>
-            <tr>
-                <td>2010-14</td>
-                <td>The Company</td>
-                <td>Barcelona</td>
-                <td>Worker</td>
-                <td>Many responsabilities</td>
-            </tr>
-            <tr>
-                <td>2014-17</td>
-                <td>The Other</td>
-                <td>Barcelona</td>
-                <td>Boss</td>
-                <td>Many more responsabilities</td>
-            </tr>
-        </table>
-    </section>
-    
-    <hr>
-
-    <section>
-        <h2>Education</h2>
-        <table>
-            <tr>
-                <th>Years</th>
-                <th>Educator</th>
-                <th>Location</th>
-                <th>Topics</th>
-            </tr>
-            <tr>
-                <td>2000-05</td>
-                <td>Life</td>
-                <td>World</td>
-                <td>Laughing</td>
-            </tr>
-            <tr>
-                <td>2005-10</td>
-                <td>University</td>
-                <td>Barcelona</td>
-                <td>Learning</td>
-            </tr>
-        </table>
-    </section>
-
-    <hr>
-
-    <section>
-        <h2>Hobbies and Interests</h2>
-        <ol>
-            <li>Lindy Hop dancing</li>
-            <li>Meditation</li>
-            <li>Gastronomy</li>
-        </ol>
-    </section>
+    <main>
+        
+        <section class='introduction section' >
+            <div class='introduction__image'>
+                <img id= 'picture' src="https://placekitten.com/280/201">
+            </div>
+            <div class='section__text'>
+                <p>Hello and Wellcome</p>
+                <h2 class='section__title'>I'm ANA SANJUÁN</h2>
+                <address>
+                    <p>Email: <a href= "mailto:lorem@lorem.com">lorem@lorem.com</a></p>
+                    <p>Address: Lorem Ipsum Street 1, Barcelona<br></p>
+                    <p>Phone: +00-5555-555</p>   
+                </address>
+                <nav>
```

Comment:
Is necessary the tag nav? Is it a navigation menu? 
Anyway, I think the nav tag includes a list ul. -->  https://www.w3schools.com/css/css_navbar.asp

Reply:
thanks for the comment. 
I checked the FM and I see now that nav is only for navigation inside your page, I didn´t know it before! :)
Also, as you suggested, I will use a list here